dozer-log/src/replication/mod.rs

```diff
@@ -93,12 +94,23 @@ impl Log {
     }
 
     pub async fn new(options: LogOptions, log_dir: String, readonly: bool) -> Result<Self, Error> {
-        let (storage, prefix) = create_log_storage(options.storage_config, log_dir).await?;
+        let (storage, mut prefix) = create_log_storage(options.storage_config, log_dir).await?;
+        if !readonly {
```

Comment:
Can you add a comment here explaining what the marker file does, with a `TODO:` or `XXX:` tag or something to revisit this in the future?

Reply:
Added. I'll resolve and merge. Thanks!